docs/interactions/structured-messages/webview.md

```diff
@@ -0,0 +1,51 @@
+
+The webview is a small window that sits atop the chat. It can be customized in size through the `webview_height` parameter.
+
+## Use Case
+
+The webview is not just a simple window opening, it can be used to display a custom survey, to select a room or a seat on a train or even a more attractive way to select a subscription. For each action taken on the page, a custom response can be sent for the agent side, that can confirm a seat selection or express gratitude to a client indicating satisfaction.
+
+## Recommended Usage
+
+It should be noted that no default picker or page is provided and this feature is intended to be customizable, not out of the box. No default response after the customer has finished taking actions on the page is provided either.
+
+What is recommended is setting the conversation's `data-id` inside of the url query string in order to provide a custom page taylored to the customer (e.g displaying a confirmation instead of a choice the second times the customer clicks on it), avoiding him doing several times the procedure.
+
+You can also use this `id` to send a `post` request to Engage Digital API to send a message to the customer after he has taken all actions on the page to confirm or give a summary of his choices. Passing any parameter the client has already given the agent through the query string is possible, such as a ticket number or an email.
+
+## Integration
+
+The page on the webview should be hosted on your side, and setup to accept display on other domains. Check [Webview SDK](../../structured-messages/webview-sdk) for precisions on the integration.
+
+## Channels supporting this feature
+
+* **<span style="color:green">Yes</span>**: Supported
+* **<span style="color:orange">Not yet</span>**: Supported by channel but not by Engage Digital yet
+* N/A: Not supported by channel
+
+  |  | **webview** | **webview_height** |
+  |-|-|-|
+  | **Apple Business Chat** | N/A | N/A |
+  | **Engage Android Messaging** | **<span style="color:orange">Not yet</span>** | **<span style="color:orange">Not yet</span>** |
+  | **Engage iOS Messaging** | **<span style="color:orange">Not yet</span>** | **<span style="color:orange">Not yet</span>** |
+  | **Engage Web Messaging** | **<span style="color:green">Yes</span>** | **<span style="color:green">Yes</span>** |
+  | **Facebook Messenger** | **<span style="color:orange">Not yet</span>** | **<span style="color:orange">Not yet</span>** |
+  | **Google Business Messages** | N/A | N/A |
+  | **Source SDK** | **<span style="color:orange">Not yet</span>** | **<span style="color:orange">Not yet</span>** |
+  | **Twitter Direct Messages** | **<span style="color:orange">Not yet</span>** | N/A |
+  | **Viber** | **<span style="color:orange">Not yet</span>** | N/A |
```

Comment:
We don't really plan on supporting it on Viber, nor on Twitter. So you can N/A them.